docs/components/modeler/what-is-camunda-modeler.md

```diff
@@ -0,0 +1,24 @@
+---
+id: what-is-camunda-modeler
+title: What is Camunda Modeler?
+---
+
+Any executable process needs a **BPMN diagram** designed and configured beforehand.
+
+A BPMN diagram is used to visually outline the structure and flow of a process. As a result, the process can be more easily understood by various stakeholders.
+
+In tandem, different events and implementation details (such as the conditions within a gateway, or the specifications of a service task) must be configured in the model so the workflow engine understands what must executed once the process reaches a certain task.
+
+Camunda offers three tools to design your diagrams and implement them:
+
+- [Cawemo](https://cawemo.com/)
+- [Cloud Modeler](./cloud-modeler/launch-cloud-modeler.md)
+- [Camunda Modeler](./camunda-modeler/install-the-modeler.md)
+
+**Cawemo** focuses on the design phase and provides options for collaborative work on a BPMN model.
+
+**Cloud Modeler** and **Camunda Modeler** support the technical parameterization of the models. It's possible to design the model in Cawemo, and continue working with Cloud Modeler or Camunda Modeler.
```

Comment:
And then reordering this to match.

Reply:
Please rename `parameterization` with `implementation` (adding implementation related meta-data, making the process executable, ...).